applications.json

```diff
@@ -1,5 +1,21 @@
 {
   "applications" : [
+    {
+      "short_description" : "cross platform e-book manager.",
+      "categories" : [
+        "Utilities"
+      ],
+      "repo_url" : "https://github.com/kovidgoyal/calibre",
+      "title" : "calibre",
+      "icon_url" : "",
```

Comment:
Please add "icon_url" = https://github.com/kovidgoyal/calibre/blob/master/resources/images/lt.png .

Reply:
I make it.